django/db/models/sql/compiler.py

```diff
@@ -2112,6 +2126,39 @@ def execute_sql(self, result_type):
                 is_empty = False
         return row_count
 
+    def execute_returning_sql(self, returning_fields):
+        """
+        Execute the specified update and return rows of the returned columns
+        associated with the specified returned_fields if the backends supports
+        it.
+        """
+        if self.query.get_related_updates():
+            raise NotImplementedError(
+                "Update returning is not implemented for queries with related updates."
+            )
+
+        if (
+            not returning_fields
+            or not self.connection.features.can_return_rows_from_update
+        ):
+            row_count = self.execute_sql(ROW_COUNT)
+            return [()] * row_count
```

Comment:
In cases where no returning fields are specified use the usual _row count_ path and return a list of empty tuples for each updated row.